mockk/jvm/src/test/kotlin/io/mockk/junit4/MockKRuleTest.kt

```diff
@@ -0,0 +1,85 @@
+@file:Suppress("UNUSED_PARAMETER")
+
+package io.mockk.junit4
+
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.impl.annotations.SpyK
+import io.mockk.verify
+import org.junit.Rule
+import org.junit.jupiter.api.Test
```

Comment:
This should actually be `org.junit.Test`

Reply:
Changed!